dogs-initial/src/main/java/com/lambdaschool/dogsinitial/controller/DogController.java

```diff
@@ -36,4 +45,13 @@
                 findDogs(d -> d.getBreed().toUpperCase().equals(breed.toUpperCase()));
         return new ResponseEntity<>(rtnDogs, HttpStatus.OK);
```

Comment:
It should be noted that expectations for returning an empty list may vary - some front end developers might expect an empty array, but for other cases, a `ResourceNotFoundException` (or some other type of exception) might be appropriate.